src/components/Card/native/Card.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Card = styled.View`
+  background: ${(props) => props.theme.color[props.color]};
+  border-radius: ${(props) => props.theme.borderRadius};
+  elevation: ${(props) => props.elevation};
```

Comment:
Does this work? as far as I know, View does not accept elevation as a style.

Reply:
It is platform specific, only supports Android as of now.
For iOS there is shadow prop which does the same.